style.css

```diff
@@ -0,0 +1,163 @@
+@import url("https://fonts.googleapis.com/css2?family=Merriweather&display=swap");
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Futura", Tahoma, Geneva, Verdana, sans-serif;
+  color: #333c57;
+}
+
+body {
+  background: #60789b;
+}
+
+h1 {
+  font-family: "Merriweather", serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 2.5em;
+  color: #afc1d7;
+  padding: 50px 0;
+  text-align: center;
+  background: #333c58;
+}
+
+h2 {
+  margin: 10px 0;
+  font-family: "Merriweather", serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 1.5em;
+  color: #333c57;
+}
+
+.sort-buttons {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin: 0 20px;
+}
+
+#library-container {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 10px;
+  padding: 20px;
+  background: #60789b;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: #afc1d7;
+  padding: 20px;
+  border-radius: 20px;
+  width: auto;
+}
+
+.random {
+  background: white;
+  display: grid;
+}
+
+#random-book-container {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 10px;
+  padding: 5px 20px 0 20px;
+  justify-content: center;
+}
```

Comment:
this element is changing height based on the content, which should be avoided in this case. It's better to set the same `height` for all cases. Also make sure the elements inside every card are placed consistently and don't have a different position based on the size of other elements (in this case the change of description length is determining their positions). So I recommend you set a `height` for the description `div` that is the same for all cases 👍